intro-to-R-tidyverse/03-intro_to_r_tidyverse_exercise.Rmd

```diff
@@ -0,0 +1,160 @@
+                                                                                                                                                                                                                                                                                                                                                                                ---
+title: "Exercise: Clean Metadata and Prepare Gene Expression Matrix for Downstream Analysis"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+The goal of these exercises is to help you get familiar with using tidyverse functions to manipulate data. 
+
+Here, 
+TODO: Determine suitable experiment and write about it
+
+### Setup 
+Before we start the exercises, we need to create the necessary directories and loading the necessary libraries.
+
+#### Creating necessary directories
+We need to create a directory to store the results of the exercises. We have done this for you. 
+Run the code chunk below and take a look at your files pane. You should see a directory `exercise-results` created. 
+
+```{r}
+#creating required directories as needed
+if (!dir.exists("excercise-results")) {
+  # ... create a 'results' directory
+  dir.create("exercise-results")
+}
+```
+
+#### Loading libraries
+For these exercises we will need the `tidyverse` library. Go ahead and load the library in the code chunk below. Don't forget to run the code chunk once you write the command! 
```

Comment:
```suggestion
For these exercises, we will need the `tidyverse` library. Go ahead and load the library in the code chunk below.
Don't forget to run the code chunk once you write the command! 
```

Reply:
Also, our generally formatting tendencies/rules/guidelines has been starting each sentence on a new line. I *think* we intend to keep following those rules in the exercise notebooks, but we particularly wanna follow that guideline for lines that end up being larger than 80 characters.

Reply:
Is there a reason for the 80 characters per line guideline? Just curious.

Reply:
That's what the style guides that we follow say https://google.github.io/styleguide/Rguide.xml#linelength